tests/skustomize.bats

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bats
+bats_load_library "bats-support"
+bats_load_library "bats-assert"
+bats_require_minimum_version 1.5.0
+
+setup_file() {
+  GIT_ROOT=$(git rev-parse --show-toplevel)
+  export TEST_DIR="$GIT_ROOT/tests"
+  export SKUST_BIN="$GIT_ROOT/skustomize"
+  export SOPS_AGE_KEY_FILE="$TEST_DIR/age/key.txt"
+  TEMPDIR=$(mktemp -d)
+  export TEMPDIR
+  cp "$TEST_DIR"/secrets.yaml "$TEMPDIR"
+  cd "$TEST_DIR"
```

Comment:
⚠️ **[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use 'cd ... || exit' or 'cd ... || return' in case cd fails. [SC2164](https://github.com/koalaman/shellcheck/wiki/SC2164)